### DISCUSSION TITLE
Fivetran load_assets_from_fivetran_instance question

### DISCUSSION THREAD

**Discussion by rmukunda** [dagster-io/dagster#18530] [Q&A] [answered]
I am loading fivetran connectors using load_assets_from_fivetran_instance. I see that these connectors are getting syncd one after the other when I see the Timed View in runs. Is there a way to run these syncs in parallel from dagster?

**Comment by yuhan** [comment, marked as answer]
You can change the max concurrent limit for assets in a single run: https://docs.dagster.io/guides/limiting-concurrency-in-data-pipelines#limiting-overall-concurrency-in-a-job
